packages/atomic/cypress/integration/facets/color-facet/color-facet.cypress.ts

```diff
@@ -176,6 +177,7 @@ describe('Color Facet Test Suites', () => {
         describe('when selecting a search result', () => {
           function setupSelectSearchResult() {
             setupSearchFor();
+            AnalyticsTracker.reset();
```

Comment:
In some cases where two analytics of the same type are sent in a row, tests are gonna fail since they will assert on the previously logged analytics rather than the upcoming one. There's two ways to prevent this.
1. Clearing old recorded analytics
2. Making AnalyticsTracker more specific to leverage Cypress's retry mechanism